extensions/one-time-password/package.json

```diff
@@ -5,6 +5,10 @@
   "description": "Generate time-based one-time passwords",
   "icon": "icon.png",
   "author": "lachero",
+  "version": "1.1.0",
```

Comment:
The `version` field is unnecessary and nonsense for Raycast extension. Let's remove it.

Reply:
then how do I refer to the update in the changelog?

Reply:
Just update the changelog.

Reply:
sure